projects/dxilconv/lib/ShaderBinary/ShaderBinary.cpp

```diff
@@ -297,7 +298,7 @@ void InitInstructionInfo()
 
 void CShaderCodeParser::SetShader(CONST CShaderToken* pBuffer)
 {
-    m_pShaderCode = (CShaderToken*)pBuffer;
+    m_pShaderCode = const_cast<CShaderToken*>(pBuffer);
```

Comment:
As with currentToken and ShaderTokenEnd above, I think `m_pShaderCode` can and should be made const.